server/src/routes/example.route.js

```diff
@@ -1,11 +0,0 @@
-const express = require('express');
-const router = express.Router();
-const exampleController = require('../controllers/example.controller');
-
-/** @route      GET /api/example
- *  @desc       fetch single example
- *  @access     Public
- */
-router.get('/', exampleController.getExample);
-
-module.exports = router;
```

Comment:
Kindly don't delete this file as this is for reference of conventions which you have not adopted throughout and would recommend if you could follow this.